Package.swift

```diff
@@ -25,6 +25,18 @@ let package = Package(
     ],
     dependencies: [],
     targets: [
+        .binaryTarget(
```

Comment:
I've bundled these within the repo under `/frameworks` due to the 3ds sdk required authentication to access.
Note that the first 2 here and dependencies for the 3ds sdk